src/main/res/values/strings.xml

```diff
@@ -226,8 +226,8 @@
     <string name="action_hide">Hide</string>
     <string name="action_unsave">Unsave</string>
     <string name="action_unhide">Unhide</string>
-    <string name="action_external">View in External Browser</string>
-    <string name="action_external_short">In Browser</string>
+    <string name="action_external">View in External Web Browser</string>
+    <string name="action_external_short">In Web Browser</string>
```

Comment:
Unnecessary extra word (applies to all changes of this type)

Reply:
The difference between a translation stating web-browser and file-browser is that the ambiguity turns into being 100% wrong 100% of the time.